docs/cita/rpc-guide/rpc.md

```diff
@@ -393,8 +393,8 @@ Object - 回执对象
 * transactionIndex: `Quantity` - 交易 `index`
 * blockHash: `Data32` - 交易所在块的块哈希
 * blockNumber: `Quantity` - 交易所在块的块高度
-* cumulativeQuotaUsed: `Quantity` - 虚拟机执行交易消耗 quota 数量
-* quotaUsed: `Quantity` - 交易消耗的 quota 总量
+* cumulativeQuotaUsed: `Quantity` - 交易消耗的 quota 总量
+* quotaUsed: `Quantity` - 虚拟机执行交易消耗 quota 数量
```

Comment:
```rs
pub struct Executed {
    /// True if the outer call/create resulted in an exceptional exit.
    pub exception: Option<evm::Error>,

    /// Gas paid up front for execution of transaction.
    pub gas: U256,

    /// Gas used during execution of transaction.
    pub gas_used: U256,

    /// Gas refunded after the execution of transaction.
    /// To get gas that was required up front, add `refunded` and `gas_used`.
    pub refunded: U256,

    /// Cumulative gas used in current block so far.
    ///
    /// `cumulative_gas_used = gas_used(t0) + gas_used(t1) + ... gas_used(tn)`
    ///
    /// where `tn` is current transaction.
    pub cumulative_gas_used: U256,

    /// Vector of logs generated by transaction.
    pub logs: Vec<LogEntry>,

    /// Addresses of contracts created during execution of transaction.
    /// Ordered from earliest creation.
    ///
    /// eg. sender creates contract A and A in constructor creates contract B
    ///
    /// B creation ends first, and it will be the first element of the vector.
    pub contracts_created: Vec<Address>,
    /// Transaction output.
    pub output: Bytes,
    /// The trace of this transaction.
    pub trace: Vec<FlatTrace>,
    /// The VM trace of this transaction.
    pub vm_trace: Option<VMTrace>,
    /// The state diff, if we traced it.
    pub state_diff: Option<StateDiff>,
    /// Transaction sender account nonce
    pub account_nonce: U256,
}
```

Reply:
ok, I'll check it again.